src/test/resources/org/rust/ide/annotator/fixtures/type_parameters.rs

```diff
@@ -3,6 +3,6 @@ trait <info>MyTrait</info> {
     fn <info>some_fn</info>(&self);
 }
 
-struct <info>MyStruct</info><<info>N<info>:</info> ?<info>Sized</info>+<info>Debug</info>+<info><info>MyTrait</info></info></info>> {
+struct <info>MyStruct</info><<info>N</info>: ?<info><info>Sized</info></info>+<info><info>Debug</info></info>+<info><info><info>MyTrait</info></info></info>> {
```

Comment:
No, I believe `traitRefs` are highlighted twice here, while they shouldn't be.  I think that if you remove completely [this `forEach`](https://github.com/intellij-rust/intellij-rust/pull/503/files#diff-bc8f1651165bb2662ad415a152f3f9eaR61) then the result will be just right.